src/v2/Apps/Example/__tests__/ExampleApp.jest.tsx

```diff
@@ -0,0 +1,74 @@
+import { ExampleAppFragmentContainer } from "../ExampleApp"
+import { setupTestWrapperTL } from "v2/DevTools/setupTestWrapper"
+import { graphql } from "react-relay"
+import { ExampleApp_Test_Query } from "v2/__generated__/ExampleApp_Test_Query.graphql"
+import { MockBoot } from "v2/DevTools"
+import { screen } from "@testing-library/react"
+import { findMetaTagBySelector } from "v2/DevTools"
+
+jest.unmock("react-relay")
+
+const { renderWithRelay } = setupTestWrapperTL<ExampleApp_Test_Query>({
+  Component: props => {
+    return (
+      <MockBoot>
+        {/* @ts-expect-error PLEASE_FIX_ME_STRICT_NULL_CHECK_MIGRATION */}
```

Comment:
You'd just use `@ts-ignore` if you wanted to ignore this. This form is from the migration. To actually fix this problem you can just do something like `if (!props.system) return null` or whatever it's complaining about

Reply:
(Not that it matters a ton in this test code 🤷 )

Reply:
thanks for the hint 🙏